src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -1009,13 +1009,16 @@ def _create_identity_instance(cli_ctx, authority, tenant_id=None, client_id=None
     from .telemetry import set_broker_info
     set_broker_info(enable_broker_on_windows)
 
+    # On WSL, use core.enable_broker_on_wsl=true to use broker (WAM)
+    enable_broker_on_wsl = cli_ctx.config.getboolean('core', 'enable_broker_on_wsl', fallback=False)
```

Comment:
Telemetry for `enable_broker_on_wsl` will be added by https://github.com/Azure/azure-cli/pull/31275.

Sending telemetry during development will pollute the telemetry as the feature is not ready yet.